### PR TITLE
chore(windows): version.rc use macros instead of mkver ✈

### PR DESCRIPTION
**Pull request by mcdurdin**
This cleanup moves to using `#include "keymanversion.h"` in the various version.rc files instead of rewriting the version fields from a version.in template file with mkver, thus reducing our dependence on mkver -- and eliminating the version.in files altogether. There should be no material difference between the functionality in the two build processes.

Note: follow-up PR #6022 centralizes a few more of the values in the version.rc resources.

@keymanapp-test-bot skip

### CODE REVIEW

File: windows/src/developer/kmndebug/version.rc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+      BLOCK "0C0904E4"
+      BEGIN
+        VALUE "CompanyName", "SIL International\0"
+        VALUE "FileDescription", "Keyman KMN Debug Library\0"
+        VALUE "FileVersion", "15.0.78.0\0"
+        VALUE "InternalName", "KMNDEBUG\0"
+        VALUE "LegalCopyright", "� SIL International\0"
```
Comment (darcywong00) [inline windows/src/developer/kmndebug/version.rc:18]:
```suggestion
        VALUE "LegalCopyright", "\xA9 SIL International\0"
```

File: windows/src/test/manual-tests/test_i3619/i3619tip/i3619tip/i3619tip/version.rc [hunk 2]
```diff
@@ -0,0 +1,30 @@
+1 VERSIONINFO
+  FILEVERSION		9,0,421,0
```
Comment (darcywong00) [inline windows/src/test/manual-tests/test_i3619/i3619tip/i3619tip/i3619tip/version.rc:2]: Should this use macros?

File: windows/src/developer/kmndebug/version.rc (hunk 1 above)
Comment (mcdurdin) [reply windows/src/developer/kmndebug/version.rc:18]: Good catch. I will fix the version.rc issues in #6022 given there are other updates to version.rc there anyway!

**Review by darcywong00 — APPROVED** [review]
lgtm

My review comments will be addressed in #6022

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.166-alpha](https://keyman.com/downloads/releases/alpha/15.0.166)

**Pull request merged**